install.sh: preload some images

vdemeester opened the pull request:
- the golang image used for demos
- the kobw-builder image used for demos
- any knative to-be-cached images

This should move the 1st build time really close (if not the same as) the next builds (aka ~10s)

cc @matzew @jcrossley3 @markusthoemmes 

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

markusthoemmes commented on etc/scripts/install.sh:
```diff
@@ -129,5 +129,13 @@ oc -n knative-serving get cm config-controller -oyaml | sed "s/\(^ *registriesSk
 
+eval "$(minishift docker-env)"
+docker image pull centos/go-toolset-7-centos7:latest
+docker image pull vdemeester/kobw-builder:0.1.0
```
Should this be `0.1.1` now?

vdemeester replied:
@markusthoemmes indeed !

vdemeester committed:
install.sh: preloud some images

- the golang image used for demos
- the kobw-builder image used for demos
- any knative to-be-cached images

Signed-off-by: Vincent Demeester <vincent@sbr.pm>

jcrossley3 commented:
We can put this in for the demo, but ultimately i'd like this script to be a way to install knative on any openshift and we probably don't need the `kobw-builder` image installed for those.

jcrossley3 commented:
Per discussion in slack, i'm gonna close this with the understanding you'll add it to the demos repo, thanks!
